tests/ignite/metrics/test_confusion_matrix.py

```diff
@@ -55,145 +55,97 @@ def test_multiclass_wrong_inputs():
 
 def test_multiclass_input_N():
```

Comment:
Can we refactor all three `multiclass_input_` functions? we can just implement one function `test_multiclass_input` that has all three types of test cases.